### ISSUE TITLE
Outreach to contacts beyond GSA

### ISSUE THREAD

**Issue by grugnog** [GSA/open-product-accessibility-template#21]
- [ ] Determine publications
- [x] Key themes

Publications (ideas)

- [ ] FCW
- [ ] GCN
- [ ] GovMatters
- [ ] FedScoop
- [ ] StateScoop
- [x] GovExec
- [ ] NextGov

Who else?
- [x] W3C
- [x] Accessibility community (IAAP)
- [ ] European Commission 
- [x] Vendor community
- [x] Web-a11y Slack (#mr-vpat channel)

**Comment by mgifford** [comment]
Wanted to link to our [internal list of media](https://docs.google.com/spreadsheets/d/1AvmXi2xJ_e32XxlrzYsGwyDbwW0F2XcPeowRDCiieTw/edit#gid=1587038171). 

The key themes above should include the elements of the [short summary doc](https://docs.google.com/document/d/1zFXOnp7lSdxHx67eXM4P3oBEykRwsRsYQP6oM-oj13E/edit) but can focus on the vision:
- Buyers can more easily compare solutions
- Managers can finally evaluate what interfaces may limit their staff
- Executives can better understand the risks of present digital interfaces
- Vendors can provide government better transparency into product accessibility conformance

and the solution:
- A machine-readable semantic version that separates content from the presentation layer
- An intuitive user interface that empowers software vendors to generate an ACR that can be written in open standards, allowing the product to be extended 
- An enterprise view where multiple ACRs to be organized and consistently presented
- A function that compares ACR versions and tracks changes over time
- A built-in validation layer to ensure data quality and integrity
